src/components/layout/Page.jsx

```diff
@@ -18,25 +19,30 @@ const useStyles = makeStyles((theme) => ({
  * Dashboard > Page
  */
 const Page = (props) => {
-  const {id, children, orientation} = props;
+  const {id, children, orientation, errorStatus, errorMessage} = props;
   const classes = useStyles();
 
   // Variables
   const pageDirection = orientation == 'columns' ? 'row' : 'column';
 
   // Configure flex versus parent container
+  const elements =
+    errorStatus != null ? <Error status={errorStatus} message={errorMessage} /> : children;
+
   return (
     <Box id={id} sx={{flexGrow: 1}}>
       <Grid container spacing={2} direction={pageDirection} className={classes.pageLayout}>
-        {children}
+        {elements}
       </Grid>
     </Box>
   );
 };
 
 Page.defaultProps = {
   id: 'page',
-  orientation: 'columns'
+  orientation: 'columns',
+  errorStatus: null,
+  errorMessage: null
```

Comment:
🐼 

Maybe initialise by default with a blank string `''`, in case such a value isn't passed by the callback later on down the line?